test/run-spec-wasm2c.py

```diff
@@ -147,7 +157,7 @@ def GetModulePrefixUnmangled(self, idx):
         return self.unmangled_names[idx]
 
     def GetModuleInstanceName(self, idx_or_name=None):
-        return self.GetModulePrefix() + '_instance'
+        return self.GetModulePrefix(idx_or_name) + '_instance'
```

Comment:
Is this related?

Reply:
Yes, otherwise it's not possible to call GetModuleInstanceName on an idx that isn't the current idx. (Which we need to do now that we're declaring all the module instance structs upfront.) Probably this method should always have been passing the `idx_or_name` argument through.